ecf/global_det/plots/jevs_global_det_wave_grid2obs_plots.ecf

```diff
@@ -0,0 +1,91 @@
+#PBS -N jevs_global_det_wave_g2o_plots
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=00:15:00
+#PBS -l place=vscatter,select=1:ncpus=128:mem=500G
+#PBS -l debug=true
+#PBS -V
+
+set -x
+
+#%include <head.h>
+#%include <envir-p1.h>
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS_save
+
+############################################################
+# read version file and set model_ver
+############################################################
+versionfile=$HOMEevs/versions/run.ver
+. $versionfile
+export evs_ver=$evs_ver
+export model_ver=$gfs_ver
+export obsproc_ver=$obsproc_ver
+
+############################################################
+# Load modules
+############################################################
+module reset
+#export HPC_OPT=/apps/ops/para/libs
```

Comment:
We might want to change this in your feature branch (and thus the authoritative) to 

`export HPC_OPT=/apps/ops/prod/libs`

This will keep it consistent with your working copy.